packages/table/src/cell/cell.tsx

```diff
@@ -25,13 +27,25 @@ export interface ICellProps extends IIntentProps, IProps {
 export type ICellRenderer = (rowIndex: number, columnIndex: number) => React.ReactElement<ICellProps>;
 
 export const emptyCellRenderer = (_rowIndex: number, _columnIndex: number) => <Cell />;
+export const loadingCellRenderer = () => <Cell isLoading={true} />;
+
+const CELL_CLASSNAME = "bp-table-cell";
 
 @PureRender
 export class Cell extends React.Component<ICellProps, {}> {
     public render() {
-        const { style, tooltip, className } = this.props;
-        const content = (<div className="bp-table-truncated-text">{this.props.children}</div>);
-        const classes = classNames("bp-table-cell", className, Classes.intentClass(this.props.intent));
+        const { style, isLoading, tooltip, className } = this.props;
+        const content = isLoading
+            ? renderLoadingSkeleton(CELL_CLASSNAME)
+            : (<div className="bp-table-truncated-text">{this.props.children}</div>);
```

Comment:
still need to implement this for `EditableCell`. but as we discussed, i'll first move the skeleton to be a core component so we don't have so much branching code paths in BP table